inference/generativeai/llm-workshop/lab11-llama2/meta-llama-2-70b-lmi.ipynb

```diff
@@ -0,0 +1,1060 @@
+    "## Create SageMaker compatible Model artifact,  upload Model to S3 and bring your own inference script.\n",
+    "\n",
+    "SageMaker Large Model Inference containers can be used to host models without providing your own inference code. This is extremely useful when there is no custom pre-processing of the input data or postprocessing of the model's predictions.\n",
+    "\n",
+    "SageMaker needs the model artifacts to be in a Tarball format. In this example, we provide the following files - serving.properties.\n",
+    "\n",
+    "The tarball is in the following format:\n",
+    "\n",
+    "```\n",
+    "code\n",
+    "├──── \n",
+    "│   └── serving.properties\n",
+    "```\n",
+    "\n",
+    "    serving.properties is the configuration file that can be used to configure the model server.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b50f5d44-6388-49fa-bc87-2e82bed226f3",
+   "metadata": {},
+   "source": [
+    "#### Create serving.properties \n",
+    "This is a configuration file to indicate to DJL Serving which model parallelization and inference optimization libraries you would like to use. Depending on your need, you can set the appropriate configuration.\n",
+    "\n",
+    "Here is a list of settings that we use in this configuration file -\n",
+    "\n",
+    "    engine: The engine for DJL to use. In this case, we have set it to MPI.\n",
+    "    option.model_id: The model id of a pretrained model hosted inside a model repository on huggingface.co (https://huggingface.co/models) or S3 path to the model artefacts. \n",
+    "    option.tensor_parallel_degree: Set to the number of GPU devices over which Accelerate needs to partition the model. This parameter also controls the no of workers per model which will be started up when DJL serving runs. As an example if we have a 8 GPU machine and we are creating 8 partitions then we will have 1 worker per model to serve the requests.\n",
+    "\n",
+    "For more details on the configuration options and an exhaustive list, you can refer the documentation - https://docs.aws.amazon.com/sagemaker/latest/dg/realtime-endpoints-large-model-configuration.html.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f78a58e2-d5e7-4c13-a6ff-b814cc399dad",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# define a variable to contain the model name or s3url of the location with model artefacts. DJL container can load the model from the S3 path much faster\n",
+    "model_name_or_path = \"TheBloke/Llama-2-70B-fp16\"  # or \"s3://customer_bucket_name/..\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c2691f5b-4e98-4f7a-887c-49c05bbf7a8e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#remove existing code dir if exists\n",
+    "if os.path.exists(\"code_llama2_70b_fp16\"):\n",
+    "    !rm -r code_llama2_70b_fp16\n",
```

Comment:
```suggestion
    "!rm -rf code_llama2_70b_fp16\n",
```

Reply:
Updated the notebooks with suggested changes